Features/ProfilePage.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Profile Management
+
+Background:
+    Given I am on the home page
+    When I click on the sign-in button
+    And I enter the username "user@example.com" and password "password123"
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

- E.g of above scenarios
- Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.
- 
- The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).
- 
- Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).
- 
- If some scenarios does not make sense to add you can you should know why it does not make  sense to add.

Reply:
Thank you for your feedback! This has been fixed.